downstream/assemblies/hub/assembly-hub-connect-sso.adoc

```diff
@@ -0,0 +1,10 @@
+[id="assembly-hub-connect-sso"]
+
+= Connecting {HubName} to a {RHSSO} environment
+
+To connect {HubName} to a {RHSSO} installation, configure inventory variables in the `inventory` file before you run the installer setup script.
```

Comment:
Just a thought, but why `inventory` file and not `inventory` variable?

You can  go ahead and ignore this.